src/Tribe/Aggregator/Record/Eventbrite.php

```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
 	 * @return array
 	 */
 	public static function filter_setup_do_not_override_post_status( $post_status, $event, $record ) {
-
-		if ( ! isset( $event['eventbrite']->status ) || ! isset( $record->meta['post_status'] ) ) {
-			return $post_status;
-		}
-
-		// We're only interested in the "(do not override)" post status.
-		if ( 'do_not_override' !== $record->meta['post_status'] ) {
-			return $post_status;
+                // override status if set within import
+                $status = isset( $record->meta['post_status'] ) ? $record->meta['post_status'] : $post_status;
+                if ( 'do_not_override' === $status ) {
```

Comment:
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 2 tabs, found 4

<sub>hash: 3e687dc957d4d0ac2b3df32db37ba425</sub>